### PR TITLE
Feature/homepage

### PR DESCRIPTION
**Pull request by Chandan-devs-tech**
# Milestone 1: Personal Portfolio Mobile Version

## In this milestone I was able to :

- perfect my HTML semantics best practices knowledge
- Perfect my CSS3 layouts with flexbox and position property
- Perfect my knowledge on Github flow
- Perfect my professional documentation knowledge
- Was able yo understand how to plan my HTML to come up with a desired markup

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap");
+
+body {
+  background-color: beige;
+  display: flex;
+  justify-content: center;
```
Comment (binhussen) [inline style.css:6]: - Kindly, the `<body>` tag has by default `margin: 8px;` and other tag elements may have their default `margin and padding` so reset all tag elements `margin` and `padding` by using all selector
` * {
margin: 0;
padding: 0;
} ` 👍 👍

File: style.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  width: 375px;
+  height: 730px;
+
+  /* border: 1px solid grey; */
+  background-color: white;
+}
+
+/* header {
+    border: 1px solid red;
+} */
```
Comment (binhussen) [inline style.css:19]: - Kindly, remove unused codes instead of commenting them to make your code clean 👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @Chandan-devs-tech , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
- kindly, keep sematic conversions for reality and search engines
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Chandan-devs-tech** [comment]
## MODIFICATION :

- Modified index.html file.
- Modified style.css file.

**Review by amejid — APPROVED** [review]
Hi @Chandan-devs-tech, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**